e2e/test/scenarios/embedding/embedding-questions.cy.spec.js

```diff
@@ -212,6 +212,42 @@ describe("scenarios > embedding > questions", () => {
     // eslint-disable-next-line no-unscoped-text-selectors -- deprecated usage
     cy.contains("October 7, 2023, 1:34 AM");
   });
+
+  it("should not crash on ignored HTTP status codes (1xx & 3xx) (metabase#51386)", () => {
+    cy.intercept({
+      method: "GET",
+      url: "embed/question/**.csv**",
+      headers: {
+        "x-metabase-client": "embedding-iframe",
+        "x-metabase-embedded": "true",
+      },
+    }).as("dl");
```

Comment:
nit: I'd use the unabbreviated name for full context.

```suggestion
    }).as("download");
```

or

```suggestion
    }).as("questionDownload");
```